### ISSUE TITLE
array of ignoring rules

### ISSUE THREAD

**Issue by evilebottnawi** [anandthakker/doiuse#21]
@anandthakker bump, project die?

**Comment by anandthakker** [comment]
@evilebottnawi Not dead, but I've been in a sort of passive mode -- I can respond to specific bugs, but I haven't quite had the bandwidth for real new development recently.

**Comment by sebastienbarre** [comment]
+1 would like to see that option as well, but fully understand your bandwidth issues...
Great plugin nonetheless.

**Comment by sebastienbarre** [comment]
I can work on a PR if @anandthakker is open to the idea. Add an `ignore` option that would be an array of strings that could be compared to `usageInfo.featureData.title`?

Here is what I'm doing right now:
```javascript
    require('doiuse')({
      browsers: browserList,
      onFeatureUsage: function(usageInfo) {
        var ignore = [
          'CSS Filter Effects',
          'CSS3 Box-shadow',
          'CSS3 Colors',
          'rem (root em) units'
        ];
        if (ignore.indexOf(usageInfo.featureData.title) !== -1) {
          return;
        }
        console.log(JSON.stringify([
          usageInfo.featureData.title,
          usageInfo.message
        ]));
      }
    }),
```
An `ignore` option would be cleaner. And instead of using `onFeatureUsage` I would love for `doiuse` to send its warnings using the PostCSS `result.warn()` API, so that they can be pretty printed by `postcss-reporter` (see issue #24).
Thanks

**Comment by anandthakker** [comment]
@sebastienbarre Definitely open to a PR!  Your approach sounds great, with one tweak: how about using `usageInfo.feature` (which just a key from [the css feature list here](https://github.com/anandthakker/doiuse/blob/master/data/features.js#L14) rather than `usageInfo.featureData.title`?

**Comment by sebastienbarre** [comment]
Here you go, PR #25 added

**Comment by anandthakker** [comment]
Closed via #25
